olm-catalog/serverless-operator/hack/update-manifests.sh

```diff
@@ -15,8 +15,8 @@ target_serving_file="$target_dir/operator_v1alpha1_knativeserving_crd.yaml"
 target_eventing_file="$target_dir/operator_v1alpha1_knativeeventing_crd.yaml"
 rm -rf "$target_serving_file" "$target_eventing_file"
 
-serving_url="https://raw.githubusercontent.com/knative/operator/$version/config/300-serving.yaml"
-eventing_url="https://raw.githubusercontent.com/knative/operator/$version/config/300-eventing.yaml"
+serving_url="https://raw.githubusercontent.com/knative/operator/knative-v1.0.1/config/300-serving.yaml"
+eventing_url="https://raw.githubusercontent.com/knative/operator/knative-v1.0.1/config/300-eventing.yaml"
```

Comment:
```suggestion
serving_url="https://raw.githubusercontent.com/knative/operator/knative-$version/config/300-serving.yaml"
eventing_url="https://raw.githubusercontent.com/knative/operator/knative-$version/config/300-eventing.yaml"
```

Reply:
I think it is: `https://raw.githubusercontent.com/knative/operator/knative-v${version}/config/300-serving.yaml` you need the `v` prefix as the project.yaml file has no such prefix in versions specified.